Makefile.js

```diff
@@ -668,7 +671,11 @@ target.gensite = function(prereleaseVersion) {
     const FIXABLE_TEXT = "\n\n(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) can automatically fix some of the problems reported by this rule.";
 
     // 4. Loop through all files in temporary directory
-    find(TEMP_DIR).forEach(filename => {
+    process.stdout.write("> Updating files (Steps 4-9): 0/... - ...\r");
```

Comment:
`echo` adds a `\n` to the end, which causes the status to extend to multiple lines, which doesn’t provide a very nice experience. Additionally, `echo("-n", "foo")` doesn’t work.